### PR TITLE
pass language param to OpenCage Geocoder

### PR DESCRIPTION
**Pull request by freyfogle**
follows the same logic as Nominatim which also accepts a language param to allow user to see response in their language

### CODE REVIEW

**Comment by freyfogle** [comment]
As a follow up question to this PR, how can I submit a PR for this page: https://www.traccar.org/reverse-geocoding/ so I could also show an example of setting the language? Thank you!

File: src/main/java/org/traccar/geocoder/OpenCageGeocoder.java [hunk 1]
```diff
@@ -21,16 +21,21 @@
...
 public class OpenCageGeocoder extends JsonGeocoder {
 
-    private static String formatUrl(String url, String key) {
+    private static String formatUrl(String url, String key, String language) {
         if (url == null) {
             url = "https://api.opencagedata.com/geocode/v1";
         }
         url += "/json?q=%f,%f&no_annotations=1&key=" + key;
+        if (language == null) {
+            url += "&language=en";
```
Comment (tananaev) [inline src/main/java/org/traccar/geocoder/OpenCageGeocoder.java:30]: What if we don't pass with parameter?

File: src/test/java/org/traccar/geocoder/GeocoderTest.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public void testGisgraphy() {
     @Test
     public void testOpenCage() {
         Geocoder geocoder = new OpenCageGeocoder(
-                "http://api.opencagedata.com/geocode/v1", "SECRET", 0, new AddressFormat());
+                "http://api.opencagedata.com/geocode/v1", "SECRET", "en", 0, new AddressFormat());
```
Comment (tananaev) [inline src/test/java/org/traccar/geocoder/GeocoderTest.java:44]: I think we should test with `null`, like other geocoders.

**Comment by tananaev** [comment]
Website source code is private, so you can't submit a PR to it.

File: src/test/java/org/traccar/geocoder/GeocoderTest.java (hunk 2 above)
Comment (freyfogle) [reply src/test/java/org/traccar/geocoder/GeocoderTest.java:44]: well ideally you would have many tests with all possible values, but sure, whatever you want

**Comment by freyfogle** [comment]
> Website source code is private, so you can't submit a PR to it.

ok, you should probably document geocoder.language then

**Pull request merged**